README.md

```diff
@@ -1,105 +1,51 @@
-# Ruby Music Library
+[![Build Status](https://travis-ci.org/andela-bkanyolo/checkpoint-one-music-library.svg?branch=develop)](https://travis-ci.org/andela-bkanyolo/checkpoint-one-music-library)
+[![Code Climate](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library/badges/gpa.svg)](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library)
+[![Test Coverage](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library/badges/coverage.svg)](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library/coverage)
+[![Issue Count](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library/badges/issue_count.svg)](https://codeclimate.com/github/andela-bkanyolo/checkpoint-one-music-library)
 
-## Overview
-
-You're going to be implementing a Music Library domain composed of 3 main models, `Song`, `Artist`, and `Genre` that will relate to each other and collaborate heavily. Additionally, you're going to be extracting some common functionality of those models into a module, `Concerns::Findable` and mixing that module into those classes. You'll then build a collaborating object, `MusicImporter`, that can parse a directory of MP3 files and use the filenames to create instances of `Song`, `Artist`, and `Genre`. Finally, you'll build a CLI in `bin/musiclibrary` that is powered by a `MusicLibraryController` to provide a simple CLI that let's a user browse the library of MP3s imported by song, artist, and genre.
-
-This is a complex lab with many parts, go slow, try to understand what you're trying to build holistically before starting. Read this entire README before jumping in. As you go from spec to spec, we recommend doing them in numbered order.
-
-## Concerns
-
-A quick note on the placement of Modules. It's Ruby convention to put all Modules in a `concerns` folder and to be namespaced like this `Concerns::ModuleName`.
-
-## Instructions
-
-## `Song`, `Artist`, and `Genre` Basics
-
-The first thing to do is get the basics of the main models working. Each model has almost the exact same basic requirements, so once you make `001_song_basics_spec.rb` pass by building the `Song` class, the `Artist` and `Genre` basic specs will go fast.
-
-The requirements of each model is that they can accept a name upon initialization and set that property correctly. The `name` property should be readable and writeable by the object.
 
-```ruby
-Song.new("Blank Space").name #=> "Blank Space"`
-```
+# Fake Player
 
-Additionally, each class should contain a class variable `@@all` that is set to an empty array and is prepared to store all saved instances. This class variable should be accessible via the class method `.all`.
+## Overview
 
-```ruby
-Song.all #=> []
+This is a Command Line Application that reads from the specified path and then 
+imports all the mp3 files into its library. 
+
+## Available Commands
+
+- list songs - List all the songs in the music library
```

Comment:
#### Code Quality - Nitpick
'list songs' and other commands should be markdown annotated so as to look like commands